assets.md

```diff
@@ -47,6 +47,12 @@ but these individual OpenTelemetry members have been given rights to publish und
 * [@jack-berg](https://github.com/jack-berg)
 * [@trask](https://github.com/trask)
 
+### PyPI
+
+Link: https://pypi.org/user/opentelemetry/
+
+- Owner: [@codeboten](https://github.com/codeboten)
```

Comment:
Could we have a backup here, e.g., a user shared among the GC and stored in your 1Password account?

Reply:
The username/password is shared in 1Password. What isn't shared is the MFA on the account. I've shared the recovery codes w/ @ocelotl